### PR TITLE
BI-20: khanhvnv - Implement the simple system for managing user accounts

### PR DESCRIPTION
**Pull request by khanhvnv**
## Exercise

* [Exercise-20](https://github.com/NT-Line-team/Behave_test/issues/20)

## Overview
A system has 2 features that allow user to Register and Login
#20 
Create the above system as simple as possible with the following workflow:

Register account
Login
End
 
### Docs
- [ ] No Doc Changes.

### CODE REVIEW

File: python_basic/khanhvnv/BI_20_KhanhVu_managing user accounts simple system.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	fsuccess = 0
+	name = raw_input('Login - Enter username: ')
+	passwrd = raw_input('Login - Enter password: ')
+	
+if fsuccess == 1:
+	print "Welcome " + fullname +"!"
+else:
+	print "The maximum number of login attempts has been reached."
+	print "You could not login anymore."
+
```
Comment (hungphamviet) [inline python_basic/khanhvnv/BI_20_KhanhVu_managing user accounts simple system.py]: This file `BI_20_KhanhVu_managing user accounts simple system.py` does not conform with Python naming convention for file name. See [here](https://www.python.org/dev/peps/pep-0008/#package-and-module-names).

Comment (hungphamviet) [inline python_basic/khanhvnv/BI_20_KhanhVu_managing user accounts simple system.py]: We also have issue with business. See my tests below:

```
hungpv@Hungpv-HP khanhvnv (khanhvnv-exercise20) $ python ./BI_20_KhanhVu_managing\ user\ accounts\ simple\ system.py 
Register account
Registration - Please input username: hungpv
Registration - Please input password: hungpwd
Registration - Please input fullname: Hung Pham
Do you like to continue?(Y/N)Y
Registration - Please input username: duynguyenx
Registration - Please input password: duypwd
Registration - Please input fullname: Duy Nguyen
Do you like to continue?(Y/N)N
Login
Login - Enter username: hungpv
Login - Enter password: duypwd
Try again!
Login - Enter username: duynguyenx
Login - Enter password: duypwd
Try again!
Login - Enter username: hungpv
Login - Enter password: hungpwd
The maximum number of login attempts has been reached.
You could not login anymore.
```

**Comment by khanhvnv** [comment]
Hi Hung,

I updated code. Could you please help to review again?

Many thanks,
Khanh.

**Comment by hungphamviet** [comment]
Hi @khanhvnv ,

I see there are 2 files in the PR and I don't know which is for me to verify. Please make sure that only one file per practice.

Moreover, please shorten file name, such as, `user_management.py`.
Thanks.

**Comment by hungphamviet** [comment]
Merged #47 into `develop`

**Pull request merged**